bluemira/geometry/tools.py

```diff
@@ -932,10 +932,13 @@ def save_cad(
         components to save
     filename: str
         Full path filename of the STP assembly
-    scale: float, default = 1.0
-        The scale in which to save the Shape objects
+    formatt
+        file format to save as
+    kwargs
+        arguments passed to cadapi save function
     """
-    save_as_STP(get_properties_from_components(components, "shape"), filename, scale)
```

Comment:
So this removes the "legacy" method from the user-facing API. Question: do we want to continue to expose the legacy method? This has the potential to break some workflows.

Reply:
From our basic saving yes, it is still possible to use some interface to the legacy save STP because one the kwargs is legacy (see https://github.com/FreeCAD/FreeCAD/blob/431fafc5337976c43bf50aae7bc75b46ee8037c7/src/Mod/Import/Gui/AppImportGuiPy.cpp#L559). It will use the new interface but the old export method.

I havent documented that but I could?

Alternatively I could add another file type (eg STEP_LEGACY) that uses our old interface which is the `exportSTEP()` method on any underlying Part object.

Reply:
Also I havent removed the save_as_STP function so as long as it is a list of `BluemiraGeo` objects that will still work

Reply:
I don't mind either way, but check with the STEP guys JIC